### PR TITLE
Plain JavaScript with Object

### PR DESCRIPTION
**Pull request by rotimiazeez**
# Awesome Books: Plain JavaScript with Object  (Javascript DOM manipulations) 🎉 

>  In this project, you will build a basic website that allows users to add/remove books from a list. You will achieve that by using JavaScript objects and arrays. You will also need to dynamically modify the DOM and add basic events. 💪🏽 

Additional description of the project and its features. ➕ 

- Create a collection that keeps a list of books (hint: you can use an array of objects for that).
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection (hint: you can use the array filter() method).
- Display all books saved in the collection in the top part of the page.
- Make sure that when a user clicks the "Add" button:
 - [ ] A new book is added to the collection.
 - [ ]  The new book is displayed on the page.

- Make sure that when a user clicks the "Remove" button:
 - [ ] The correct book is removed from the collection.
 - [ ] The correct book disappears from the page.

## Built With ❤️ 

- HTML (HTML5 semantic tags, `ul`, `li`,  `button`)
-  Javascript (DOM manipulation, basic javascript syntax)

## Deployed Preview 🚀 
[Live Preview]()

### NOTE TO REVIEWER 😉 ❤️  

-

### CODE REVIEW

**Comment by mikenath223** [comment]
Hi , 

You made a great effort in this project, but unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that will be invalidated for this reason:
-  Please ensure all GitHub checks are run and are green.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Happy coding. Cheers!🚀🚀

If something isn't 100% clear, you can leave comments or questions in this PR thread.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **don't count** into the code reviews limit._

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+// REMOVE BOOKS
+function deleteBook(el) {
+  if (el.classList.contains('delete')) {
+    el.parentElement.remove();
+  }
+}
+
+document.querySelector('#list-books').addEventListener('click', (e) => {
+  deleteBook(e.target);
+});
```
Comment (mimipeshy) [inline js/index.js]: - [x] When a user clicks the "Remove" button, the right book is removed from the collection and the book disappears from the page.

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @Team 👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights
-  good GitHub workflow ✔️ 
- not forming any linter errors ✔️ 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Review by bellom — APPROVED** [review]
## Hi Team 👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rotimiazeez** [comment]
Hi @bellom 👋🏽,
Thanks for approving our project 👍🏽, That will be all for now 💪🏽

File: js/index.js (hunk 1 above)
Comment (rotimiazeez) [reply js/index.js]: Thanks for pointing this out. My partner said it was supposed to be as you said, but I declined. Thanks for clarifying things 🙏🏽

**Pull request merged**